wandb/catboost/__init__.py

```diff
@@ -0,0 +1,10 @@
+"""
+Compatibility catboost module.
+
+In the future use:
```

Comment:
does it mean we are going to deprecate this interface?
should we add a deprecation message here if we do?

Reply:
I suggest we add deprecation messages to all our compatibility modules in a separate PR, otherwise things will be inconsistent.

Reply:
sure... let's track this in an issue?

Reply:
Done!